packages/devtools_app/lib/src/screens/memory/shared/heap/model.dart

```diff
@@ -126,10 +126,69 @@ class ClassOnlyHeapPath {
             heapPath.objects.map((o) => o.heapClass).toList(growable: false);
   final List<HeapClassName> classes;
 
-  String asShortString() => classes.map((e) => e.className).join('/');
+  String asShortString({String? delimiter, bool inverted = false}) => _asString(
+        data: classes.map((e) => e.className).toList(),
+        delimiter: _delimeter(
+          delimiter: delimiter,
+          inverted: inverted,
+          isLong: false,
+        ),
+        inverted: inverted,
+      );
+
+  String asLongString({
+    String? delimiter,
+    bool inverted = false,
+    bool hideStandard = false,
+  }) {
+    late List<String> data;
```

Comment:
Does this need to be late? I would expect static analysis to have noticed that this is initialized unconditionally below.

Reply:
wow! indeed
thanks, i did not expect it to be possible :)